cmake/configure/configure_assimp.cmake

```diff
@@ -0,0 +1,20 @@
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2014 - 2015 by the deal2lkit authors
+##
+## This file is part of the deal2lkit library.
+##
+## The deal2lkit library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE at
+## the top level of the deal2lkit distribution.
+##
+## ---------------------------------------------------------------------
+
+#
+# Configuration for the sundials library:
```

Comment:
In your configure scripts, this should not be necessary:

~~~
       IF(COMMAND FEATURE_${_feature}_FIND_EXTERNAL)
          RUN_COMMAND("FEATURE_${_feature}_FIND_EXTERNAL(FEATURE_${_feature}_EXTERNAL_FOUND)")
        ELSE()
          FEATURE_FIND_EXTERNAL(${_feature} FEATURE_${_feature}_EXTERNAL_FOUND)
        ENDIF()
~~~

i.e., the default `FEATURE_FIND_EXTERNAL` macro does the exact same thing as the above, with the exception of setting the directory to "<system location>". Why add this?